### PR TITLE
Allow a "-j" or "--jobs" argument to "docker-php-ext-install" to speed up extension building

### LINKED ISSUE #144: docker-php-ext-install cpu core number option
Would be very nice to add an option on `docker-php-ext-install` command to set the number of cpu core to use.

That would be transmitted to the `make` command.

Ex: `make -j5`

### PR DESCRIPTION
**Pull request by tianon**
Fixes #144

### CODE REVIEW

**Comment by tianon** [comment]
Thanks for the prod, @Soullivaneuh :+1:

**Comment by tianon** [comment]
For testing this, I did `time docker-php-ext-install mysqli pdo pdo_mysql` vs `time docker-php-ext-install -j8 mysqli pdo pdo_mysql`, and got 13s vs 6s. :smile:

**Comment by yosifkit** [comment]
LGTM

**Comment by Soullivaneuh** [comment]
Thank you! But I don't think this is useful for `install` and `clean`. This take effect only for compilation AFAIK.

When this patch will be available from DockerHub? I'm pretty new with this system, I don't know how it works. :-)

**Comment by tianon** [comment]
Yeah, the addition to "install" and "clean" was just for consistency; it
probably won't help, but it's harmless unless the Makefile is dangerously
poorly-coded (which I'd be very surprised to find PHP's to be). :smile:

As noted in https://github.com/docker-library/php/blob/master/README.md,
this won't hit the Hub until a PR is made over in
https://github.com/docker-library/official-images with an update.

**Comment by Soullivaneuh** [comment]
> this won't hit the Hub until a PR is made over in
https://github.com/docker-library/official-images

So you're telling that I should submit the same ph on docker-library/official-images? And then, this will be automatically available?

Thanks.

**Comment by yosifkit** [comment]
Thanks for being willing but you don't have to make the PR.  I got this https://github.com/docker-library/official-images/pull/1264. :smile:

**Comment by Soullivaneuh** [comment]
Great, thanks! :+1:

File: 5.5/apache/docker-php-ext-install [hunk 1]
```diff
@@ -4,16 +4,38 @@ set -e
...
 	echo 'if custom ./configure arguments are necessary, see docker-php-ext-configure'
 	echo
 	echo 'Possible values for ext-name:'
 	echo $(find /usr/src/php/ext -mindepth 2 -maxdepth 2 -type f -name 'config.m4' | cut -d/ -f6 | sort)
 }
 
+opts="$(getopt -o 'h?j:' --long 'help,jobs:' -- "$@" || { usage >&2 && false; })"
+eval set -- "$opts"
+
+j=1
```
Comment (TimWolla) [inline 5.5/apache/docker-php-ext-install]: I wonder if this should default to `$(nproc)`.

Comment (yosifkit) [reply 5.5/apache/docker-php-ext-install]: https://github.com/docker-library/drupal/pull/154#discussion_r297898483:

>It defaults to `make`'s default (we don't explicitly `-j1` anywhere), mostly to be considerate of `php` users' machines -- for the official images though, we want to make full use of our build systems as much as possible (especially chasing the dragon of potentially faster builds), which is why we typically use `-j "$(nproc)"` in ours.

**Pull request merged**